### DISCUSSION TITLE
Feature Request - GlassBreak and SPARE as Zone Types

### DISCUSSION THREAD

**Discussion by Tipalo** [ufodone/envisalink_new#80] [Ideas]
Greetings, might be a good idea to have Spares and GlassBreak Detectors as Zone types
This might be a good icon for a broken Window:  mdi:image-broken-variant

Thanks for the good work

**Comment by ufodone** [comment]
Are you referring to the 'Show As" option when editing a zone entity?  If so, that list it something that Home Assistant core provides so if you are interested in additional options, you'd have to make a request with them.
